docs/assets/snippet-styles.css

```diff
@@ -74,12 +74,12 @@ It breaks CKEditor 5 (see how <p><code>[]</code></p> looks). */
 	padding: 0;
 }
 
-#snippet-inline-editor .ck-editor__editable {
+#snippet-inline-editor .ck-editor__editable:not(.ck-editor__nested-editable) {
```

Comment:
Can't we use `.ck-content` here? It would pick just the right element. And kinda makes sense semantically too, doesn't it? You want to align the style of the editor contents so it looks well.